### ISSUE TITLE
NPE on Archetype 22

### ISSUE THREAD

**Issue by ichirital** [adobe/aem-cloud-migration#2]
### Expected Behaviour
Running the jar on the maven archetype 22 should not cause any errors.

### Actual Behaviour
Running the jar on the maven archetype 22 causes this error:
`java.lang.NullPointerException at com.adobe.skyline.migration.parser.CustomerProjectLoader.isContentPackage(CustomerProjectLoader.java:96)`

#### Fix (which I'm not able to push)
in CustomerProjectLoader.isContentPackage() replace
```        
        String packaging = moduleXml.getElementsByTagName(MigrationConstants.PACKAGING_TAG_NAME).item(0).getTextContent();
        return packaging.equals(MigrationConstants.CONTENT_PACKAGE_PACKAGING);
```
with:
```
        NodeList packagingTags = moduleXml.getElementsByTagName(MigrationConstants.PACKAGING_TAG_NAME);
        if (packagingTags.getLength() > 0) {
            String packaging = packagingTags.item(0).getTextContent();
            return packaging.equals(MigrationConstants.CONTENT_PACKAGE_PACKAGING);
        }
        return false;
```

**Comment by ireasor** [comment]
Thanks @ichirital.  

Could you let me know how urgent this is for you?  Are you working to migrate a customer or were you just trying out the tool?

**Comment by ichirital** [comment]
Hi @ireasor it is for a customer but I am not blocked as I fixed the code locally and recompiled the jar.

**Comment by ireasor** [comment]
Good to hear.  I'm interested in any other feedback that you may have as well.  I have a backlog of planned improvements, but it is always good to have validation from real-world customer usage.

**Comment by ireasor** [comment]
Issue resolved as of release 0.2.
